kolibri/core/logger/test/test_api.py

```diff
@@ -184,3 +186,121 @@ def test_csv_download_unicode_username(self):
         for row in results[1:]:
             self.assertEqual(len(results[0]), len(row))
         self.assertEqual(len(results[1:]), expected_count)
+
+
+class MasteryLogViewSetTestCase(EvaluationMixin, APITestCase):
+    def test_summary(self):
+        for content_index, content_id in enumerate(self.content_ids):
+            content_mod = content_index % 2
+            for user_index, user in enumerate(self.users):
+                response = self.client.get(
+                    reverse("kolibri:core:masterylog-summary"),
+                    data={
+                        "content": content_id,
+                        "user": user.id,
+                    },
+                )
+                user_mod = user_index % 2
+                tries_count = (
+                    len(self.user_tries[user_index]) if content_mod == user_mod else 0
+                )
+                self.assertEqual(tries_count, len(response.data))
+                for try_index, mastery_log in enumerate(response.data):
+                    self.assertEqual(
+                        self.user_tries[user_index][try_index].id, mastery_log["id"]
+                    )
+
+    def test_diff(self):
+        for user_index, user_tries in enumerate(self.user_tries):
+            for try_index, user_try in enumerate(user_tries):
+                self.client.force_login(self.users[user_index])
```

Comment:
@rtibbles I needed this as it failed checking permissions, but I didn't run into this with any of the other tests, which makes me think the `KolibriAuthPermissionsFilter` isn't working correctly (or coder error) for the others

Reply:
Fixed it